packages/commonwealth/server/scripts/getEntitiesWithEvents.ts

```diff
@@ -0,0 +1,86 @@
+import models from '../database';
+import {QueryTypes} from "sequelize";
+import {EntityEventKind, eventToEntity, SupportedNetwork} from "chain-events/src";
+import {ChainBase, ChainNetwork} from 'common-common/src/types';
+
+async function main() {
```

Comment:
What does this script do?

Reply:
Uses the `eventToEntity` function to find the event that created the chain-entity. The problem is that a chain-entity can have several events from the same starting block so it is not possible to get the first chain-event (I call it the 'creation event') of a given chain-entity. So this script creates a table of chain-entities joined with the event that created it so that I can update root_id's for those entities.